b3lb/rest/models.py

```diff
@@ -386,9 +386,9 @@ def __str__(self):
     @property
     def endpoint(self):
```

Comment:
This does not handle single domain setups (see also #83).

Reply:
Added choice for multi, single or dual domain setup by generating 2 url links.